src/test/scala/vsys/blockchain/state/contract/channel/PaymentChannelContractDiffTest.scala

```diff
@@ -166,13 +140,13 @@ class PaymentChannelContractDiffTest extends PropSpec
     forAll(preconditionsAndPaymentChannelFunctionTest) { case (genesis: GenesisTransaction, reg: RegisterContractTransaction,
       deposit: ExecuteContractFunctionTransaction, create: ExecuteContractFunctionTransaction,
       _, _, terminate: ExecuteContractFunctionTransaction, _, _) =>
-        assertDiffEi(Seq(TestBlock.create(Seq(genesis)), TestBlock.create(Seq(reg, deposit, create))),
-          TestBlock.create(Seq(terminate))) { blockDiffEi =>
-          blockDiffEi shouldBe an[Right[_, _]]
-          blockDiffEi.explicitGet().txsDiff.contractDB.isEmpty shouldBe false
-          blockDiffEi.explicitGet().txsDiff.contractNumDB.isEmpty shouldBe true
-          blockDiffEi.explicitGet().txsDiff.portfolios.isEmpty shouldBe false
-          blockDiffEi.explicitGet().txsDiff.txStatus shouldBe TransactionStatus.Success
+        assertDiffAndState(Seq(TestBlock.create(Seq(genesis)), TestBlock.create(Seq(reg, deposit, create))),
+          TestBlock.create(Seq(terminate))) { (blockDiff, newState) =>
+          val channelId = DataEntry(Shorts.toByteArray(create.id.arr.length.toShort) ++ create.id.arr, DataType.ShortBytes).bytes
+          val contractId = reg.contractId.bytes
+          val channelStatusKey = ByteStr(Bytes.concat(contractId.arr,  Array(7.toByte), channelId))
+
+          newState.contractInfo(channelStatusKey) shouldEqual Some(DataEntry(Array(0.toByte), DataType.Boolean))
```

Comment:
similarly, it needs to use `assertDiffAndStateCorrectBlockTime`, since it also changed to expired time.